### DISCUSSION TITLE
Admin cannot share Agents

### DISCUSSION THREAD

**Discussion by leondape** [danny-avila/LibreChat#4948] [Troubleshooting]
### What happened?

Agents created by Admin don't have the sharing icons.
![Bildschirmfoto 2024-12-11 um 16 50 50](https://github.com/user-attachments/assets/0bb10e50-d986-4a71-b190-b11be4441c2a)

Also the dropdown warning is not really telling a lot what is happening actually.
![Bildschirmfoto 2024-12-11 um 16 49 26](https://github.com/user-attachments/assets/40fcf4a8-a10d-4cbd-b97d-b99b47cb735d)

### Steps to Reproduce

1. log into admin account
2. create agent
3. Admin can't share it

### What browsers are you seeing the problem on?

Microsoft Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Toggle "Allow Sharing Agents to all Users" back on, and it will resolve your issue.

At some point of the dev build, the default values for the admin role were incorrect and had that option disabled, so it would've persisted to now.

**Comment by phalcomb** [comment]
How do you find this toggle to enable sharing? I saw a reference in the docs that it could be turned on, but I couldn't any info on where exactly to enable this. Thanks.

**Comment by phalcomb** [comment]
I sorted this out, had to login to the mongodb and change my role to ADMIN and then the necessary settings appeared.

**Reply by justine-chez-gologic** [reply]
This fixed it for me too. Thank you!
